### DISCUSSION TITLE
UI suggestions in table view for filter and column icons

### DISCUSSION THREAD

**Discussion by flyingeek** [filamentphp/filament#3493] [Ideas]
- The filter icon could be filled instead of outlined when a filter is active
- The icon to collapse/show columns could be at the end of the column names row instead of being near the search button.

What do you think ?

**Comment by danharrin** [comment]
Thanks, these have been added to our roadmap - https://github.com/orgs/filamentphp/projects/2/views/1

**Reply by danharrin** [reply]
Update: I proposed the toggle button change, and people didn't like it, so not going to continue with that one.

<img width="863" alt="Screenshot 2022-08-21 at 11 43 29" src="https://user-images.githubusercontent.com/41773797/185787252-0ee30912-dd0f-4500-8fe8-b411899112b6.png">

**Reply by danharrin** [reply]
Filter indicators: #3599

**Reply by flyingeek** [reply]
I understand. I joined discord to suggest at least a title inside the popup.

![Image 2022-08-21 at 5 38 19 PM](https://user-images.githubusercontent.com/68456/185799603-3482d3de-1169-4afa-b2a2-38cacafa4f26.jpg)
